### DISCUSSION TITLE
如何分别对前端和后端 进行 打包dockerfile

### DISCUSSION THREAD

**Discussion by liminngs** [OpenCSGs/csghub#623] [Q&A]
如何分别对前端和后端 进行 打包dockerfile

**Comment by hiveer** [comment]
@liminngs 
前后台项目都有对应的 Dockerfile 文件，你可以参考。唯一需要变化的可能就是 Dockerfile 中使用到的基础镜像，您可能需要自己准备基础镜像。

有了可用的 Dockerfile，你打包就是 run 一个 docker build。

**Comment by liminngs** [comment]
> @liminngs 前后台项目都有对应的 Dockerfile 文件，你可以参考。唯一需要变化的可能就是 Dockerfile 中使用到的基础镜像，您可能需要自己准备基础镜像。
> 
> 有了可用的 Dockerfile，你打包就是 run 一个 docker build。

我看到的只有一个dockerfile  文件  请问是前后端通用这一个dockerfile文件 然后修改里面的运行命令嘛？

**Comment by Rader** [comment]
> > @liminngs 前后台项目都有对应的 Dockerfile 文件，你可以参考。唯一需要变化的可能就是 Dockerfile 中使用到的基础镜像，您可能需要自己准备基础镜像。
> > 有了可用的 Dockerfile，你打包就是 run 一个 docker build。
> 
> 我看到的只有一个dockerfile 文件 请问是前后端通用这一个dockerfile文件 然后修改里面的运行命令嘛？

后端有自己的 docker file：https://github.com/OpenCSGs/csghub-server/tree/main/docker
